### ISSUE TITLE
SwaggerRequestBodyAttribute Description have no effect in Minimal Api

### ISSUE THREAD

**Issue by akiragothick** [domaindrivendev/Swashbuckle.AspNetCore#2662]
**Minimal API**

I'm using Swashbuckle.AspNetCore 6.5.0 with Swashbuckle.AspNetCore.Annotations 6.5.0 and have added a SwaggerRequestBodyAttribute to a response class thus:

```c#
app.MapPost("/todos", ([FromBody, SwaggerRequestBody("Todo object to be created", Required = true)] Todo todo) =>
    {
        todos.Add(todo with { Id = Guid.NewGuid() });
        return TypedResults.Ok(todo);
    })
    .WithOpenApi(operation => new(operation)
    {
        Summary = "Create a todo",
        OperationId = "createTodo",
        Tags = new List<OpenApiTag> { new() { Name = "Todos" } },
        //Parameters = new List<OpenApiParameter>
        //{
        //    new OpenApiParameter{ Description ="Todo object to be created" }
        //}
    });
```

The project docs say I can use this to "to enrich the corresponding Schema metadata that's generated by Swashbuckle", but the generated swagger.json doesn't contain "MyTitle" or "MyDescription" at all, only:

```json
"post": {
    "tags": [
      "Todos"
    ],
    "summary": "Create a todo",
    "operationId": "createTodo",
    "requestBody": {
      "content": {
        "application/json": {
          "schema": {
            "$ref": "#/components/schemas/Todo"
          }
        }
      },
      "required": true
    },
    "responses": {
      "200": {
        "description": "OK",
        "content": {
          "application/json": {
            "schema": {
              "$ref": "#/components/schemas/Todo"
            }
          }
        }
      }
    }
  }
```

My configurations

```c#
builder.Services.AddSwaggerGen(options =>
{
    options.SwaggerDoc("v1", new OpenApiInfo { Title = "ChatGPT Plugin", Version = "v1" });
    options.DocInclusionPredicate((docName, apiDesc) => true);
    options.EnableAnnotations();
});
```

**Comment by Havunen** [comment]
The issue is missing the details where "MyTitle" and "MyDescription" are defined? Can you add those details please so I can have a look at this issue

**Comment by martincostello** [comment]
I believe this would be resolved by #2414.

**Comment by jgarciadelanoceda** [comment]
I have just written a PR because it was giving an exception for this matter. @martincostello this parameter is to describe the RequestBody, not the operation itself
